release/_releases_json.sh

```diff
@@ -1,6 +1,31 @@
 #!/bin/bash
 
+function _add_new_version {
+	if [ -z "${LIFERAY_RELEASE_PRODUCT_NAME}" ] || [ -z "${LIFERAY_RELEASE_VERSION}" ]
+	then
+		return
+	fi
+
+	if [[ "${LIFERAY_RELEASE_PRODUCT_NAME}" =~ ^(dxp|portal)$ ]] && [[ "${LIFERAY_RELEASE_VERSION}" =~ ^[0-9]+.([0-9])|(q[0-9])$ ]]
```

Comment:
@natocesarrego these are for the format of the input, if we try to add for example a random number like `LIFERAY_RELEASE_VERSION=5362` it will be an invalid value and won't be added to the file. Same with dxp|portal, if it is something different then the message will be `INFO "The given values are invalid"`. Should I delete this part?